cypress/integration/inventory/update-effective-location-for-item.spec.js

```diff
@@ -30,45 +30,46 @@ describe('Update the effective location for the item', () => {
           cy.getInstanceTypes({ limit: 1 });
           cy.getLocations({ limit: 1 });
           cy.getHoldingTypes({ limit: 1 });
-          InventoryHoldings.getHoldingSources({ limit: 1 }).then(holdingsSources => {
-            sourceId = holdingsSources[0].id;
-          });
         })
           .then(() => {
-            cy.createInstance({
+            InventoryInstances.createFolioInstanceViaApi({
               instance: {
                 instanceTypeId: Cypress.env('instanceTypes')[0].id,
                 title: instanceTitle,
-                source: 'FOLIO',
               },
               holdings: [{
+                holdingsTypeId: Cypress.env('holdingsTypes')[0].id,
                 permanentLocationId: Cypress.env('locations')[0].id,
-                sourceId
               }],
               items: [
-                [{
+                {
                   barcode: ITEM_BARCODE,
                   missingPieces: '3',
                   numberOfMissingPieces: '3',
                   status: { name: 'Available' },
                   permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
                   materialType: { id: Cypress.env('materialTypes')[0].id },
                 }],
-              ],
-            });
+            })
+              .then(specialInstanceIds => {
+                testInstanceIds = specialInstanceIds;
+              });
           });
         cy.login(userProperties.username, userProperties.password);
       });
     cy.visit(TopMenu.inventoryPath);
```

Comment:
Please use the path parameter of the login function to visit a page

Reply:
Done